LiteCore/Support/Logging.cc

```diff
@@ -513,33 +512,35 @@ namespace litecore {
         #endif
     }
 
+    string LogDomain::getObject(unsigned ref)
+    {
```

Comment:
Needs to lock the mutex.
Or is it guaranteed that this is only called when the mutex is already locked?